.gitlab-ci.yml

```diff
@@ -1253,6 +1255,8 @@ windows_choco_offline_7_x64:
   variables:
     ARCH: "x64"
   script:
+    - $ErrorActionPreference = "Stop"
+    - New-Item -path chocolatey\tools\ -type directory
     - copy .omnibus\pkg\*.msi chocolatey\tools\
     - docker run --rm -v "$(Get-Location):c:\mnt" 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/windows_1809_${ARCH}:$Env:DATADOG_AGENT_WINBUILDIMAGES c:\mnt\tasks\winbuildscripts\chocopack.bat
```

Comment:
I think this command should be different from the one on the online job. If I am not mistaken right now both jobs build the online version of the package (can be checked if you unzip the generated `.nupkg` and check the PowerShell script).

Reply:
The fact that's the same command is a bit error prone, though: if the MSIs don't end up there when building the offline package (as it was the case) then it builds the online one silently.

Reply:
s/was the case/still is the case/

Reply:
I rewrote it to pass an argument to the batch script and it solved the problem. Any clues as to why that might have worked @julien-lebot?